org-formation/100-shared-dns/README.md

```diff
@@ -12,16 +12,12 @@ used by the hosting service.
 
 Each wildcard cert will need to be manually verified before it can be used.
 
-### sageit.org
+### finops.sageit.org
```

Comment:
What is the motivation behind this change?  there's nothing in the referenced jira issue that describes why your doing this.

We still need `sageit.org` because we have `vpn.sageit.org`, `sc.sageit.org`, etc..

Reply:
We will still have both a `sageit.org` zone and wildcard certificate in the sageit account, this is only removing the use of the `sageit.org` zone in the admincentral account by removing the wildcard certificate from that account.

The motivation is to have friendly names for both finops lambdas, I had the wrong ticket in the subject but I've updated it to IT-2594